coinbase_HPO_test_products.py

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
roll these test cases into coinbase_tests.py

Was there a reason you made a new test file?